backend/app/api/endpoints/openapi_responses.py

```
@@ -484,21 +1121,117 @@ async def cancel_response(
-            db=db,
-            task_id=task_id,
-            user_id=current_user.id,
-            background_task_runner=background_tasks.add_task,
+    # Get task to check if it's a Chat Shell type
+    task_kind = (
+        db.query(Kind)
+        .filter(
+            Kind.id == task_id,
+            Kind.user_id == current_user.id,
+            Kind.kind == "Task",
+            Kind.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)